elfpy/bots/bot_config.py

```diff
@@ -48,6 +48,10 @@ class BotConfig(types.FrozenClass):
     # In general the bot will be more risk averse as it grows to infinity.
     # A value of 0 will usually disable it.
     risk_threshold: float = 0.0
+    # whether or not to dump state at every block
```

Comment:
this is not at every block now, right? only when reverts happen?

Reply:
changed to only in try/except

Reply:
Right, so can you fix the comment?